docs/v0.3.x/serializers.md

```diff
@@ -180,6 +182,8 @@ export default Serializer.extend({
 });
 ```
 
+*Note: This is not available yet for the JSONAPISerializer.*
```

Comment:
it's not? I thought it supports it for default includes..

Reply:
@samselikoff yes, amended thanks